index.js

```diff
@@ -45,9 +45,9 @@ function ruleset(...rules) {
             // digested. Rules run in no particular guaranteed order.
             while (nonterminals.length) {
                 const [inNode, inFlavor] = nonterminals.pop();
-                for (let rule of getDefault(rulesByInputFlavor, inFlavor, () => [])) {
+                for (const rule of getDefault(rulesByInputFlavor, inFlavor, () => [])) {
```

Comment:
This was all fallout from the new `prefer-const` rule. Since the `rule` value never changes, it didn't like the `let`.
I very vaguely recall that Firefox has/had some issues w/ `for (const foo of bar) {...}`, but I guess it depends on if somebody can use this in a browser or not.
We can also just delete/disable the `prefer-const` rule and pretend none of this ever happened.